app/views/posts/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>New Post</title>
+</head>
+<body>
+  <p>Current user: <%= @user.name %> </p>
+  <h1>Create New Post</h1>
+  <div class="container">
+    <%= form_with(model: @post, url: user_posts_path, method: :post) do |form| %>
+      <div class="form-container">
+        <div class="flex form-title">
+          <%= form.label :title, 'Title: ' %>
+          <%= form.text_field :title %>
+        </div>
+        <%= form.text_area :text, placeholder: "Enter content of the article here" %>
+        <%= form.hidden_field :author_id, value: @user.id %>
+        
+        <%= form.submit 'Create Post' %>
+      </div>
+    <% end %>
+  </div>
```

Comment:
- [ ] It would be better if we can create a post on behalf of the current user, in this situation, the button for displaying the `post form` is not showing up. I can only create post using the `rails console` command. Let us kindly fix it. The button close to the pagination button is not working as expected
![Screenshot from 2023-06-25 06-37-07](https://github.com/Swarzstein/Blog-app/assets/46388113/b0c90f3d-a731-47df-a326-521b4bdc0f20)